### PR TITLE
Added Background and Icon Color Variables + Updated Gradle + Updated Kotlin + Migrated to androidX

### PR DESCRIPTION
**Pull request by thomhurst**
Added Background and Icon Color Variables so we can customise better, with different background colors and different icon colors
Updated Gradle to 4.10
Updated Kotlin to 1.3.20
Migrated from support library to androidX

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -6,8 +6,8 @@ def configuration = rootProject.ext.configuration
 def libraries = rootProject.ext.libraries
 
 android {
-    compileSdkVersion configuration.compileSdkVersion
-    buildToolsVersion configuration.buildToolsVersion
+    compileSdkVersion 28
```
Comment (TonnyL) [inline app/build.gradle]: better if you put all the dependencies' version into one file(build.gradle of project is a good option) 😉

File: whatsnew/build.gradle [hunk 2]
```diff
@@ -37,11 +37,12 @@ android {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation "androidx.appcompat:appcompat:${libraries.supportLibraryVersion}"
-    implementation "androidx.recyclerview:recyclerview:${libraries.supportLibraryVersion}"
+    implementation "androidx.appcompat:appcompat:1.0.2"
```
Comment (TonnyL) [inline whatsnew/build.gradle]: here too.

**Comment by TonnyL** [comment]
@thomhurst  very good job 👍 !  See my comments and tell me your idea. It will be merged later.

And THANKS for your pr 🙏!

File: app/build.gradle (hunk 1 above)
Comment (thomhurst) [reply app/build.gradle]: Just changed 😄

File: whatsnew/build.gradle (hunk 2 above)
Comment (thomhurst) [reply whatsnew/build.gradle]: Unsure about these ones. AndroidX libraries aren't necessary always the same versions like the support library was.
E.g.
implementation "androidx.appcompat:appcompat:1.0.2"
implementation "androidx.recyclerview:recyclerview:1.0.0"

Comment (TonnyL) [reply whatsnew/build.gradle]: Got that. Creating a new version name (eg recyclerview) is fine.

Comment (thomhurst) [reply whatsnew/build.gradle]: Done :)

**Comment by TonnyL** [comment]
Cool!

**Comment by TonnyL** [comment]
Pretty cool, will publish it to jcenter later. Thanks for your contribution.

**Pull request merged**